cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.

```

Comment:
Consider implementing test cases to ensure the functionality of your application components such as models, views, and serializers. This will help in maintaining code quality and catching potential issues early.